packages/components/src/tooltip/index.js

```diff
@@ -71,6 +71,11 @@ class Tooltip extends Component {
 				return;
 			}
 
+			// If pressed key is escape, no further actions are needed.
+			if ( event.keyCode === 27 ) { // 27 is the keyCode for escape
```

Comment:
I would suggest to use the [`@wordpress/keycodes` package](https://github.com/WordPress/gutenberg/tree/master/packages/keycodes) to get a constant value, which negates the need for an inline code comment to explain the magic constant. It's already a dependency of `@wordpress/components`, so you should be able to just import it as such:

```js
import { ESCAPE } from '@wordpress/keycodes';

// ...
```
```suggestion
			if ( event.keyCode === ESCAPE ) {
```

Reply:
Thanks @aduth, updated this.